testsuite/src/it/java/org/graalvm/tests/integration/PerfCheckTest.java

```diff
@@ -139,7 +139,7 @@ public void testQuarkusJSONParseOnce(TestInfo testInfo) throws IOException, Inte
             Files.createDirectories(Paths.get(appDir.getAbsolutePath(), "logs"));
             assertTrue(app.buildAndRunCmds.cmds.length > 1);
 
-            if (QUARKUS_VERSION.majorIs(3) || QUARKUS_VERSION.isSnapshot()) {
+            if (QUARKUS_VERSION.compareTo(new QuarkusVersion("3.0.0")) >= 0) {
```

Comment:
Consider extracting the `3.0.0` version into a constant.

Reply:
Done